src/main/webapp/index.html

```diff
@@ -21,17 +21,16 @@
     </link>
     <script src="script.js"></script>
     <!-- API Key restricted set up on google cloud -->
-    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDFqi_fBQV7louPU53yrBCIzx0e3n0tn7o"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=MAP_KEY_API"></script>
```

Comment:
How will we deploy, just copy and paste the key here?

Reply:
Yeah. But it’s already deployed